src/N98/Magento/Command/Developer/EmailTemplate/UsageCommand.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace N98\Magento\Command\Developer\EmailTemplate;
+
+use N98\Magento\Command\AbstractMagentoCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use N98\Util\Console\Helper\Table\Renderer\RendererFactory;
+
+class UsageCommand extends AbstractMagentoCommand
+{
+    /**
+     * @var array
+     */
+    protected $infos;
```

Comment:
Thanks for the hint to dev:module:list, I'll do the correction there then after merging your PR.